dashboard/test/ui/config/scripts_json/ui-test-shared-unit.script_json

```diff
@@ -4,11 +4,13 @@
     "wrapup_video_id": null,
     "login_required": false,
     "properties": {
-      "is_migrated": true
+      "is_migrated": true,
+      "version_year": "2017"
```

Comment:
probably best to remove the `version_year`, since this lives on the unit group now?